tests/scripts/perf-prep.sh

```diff
@@ -90,9 +89,14 @@ else
     rm build.tar.gz
     popd > /dev/null
 
-    # Unzip the tests first.  Exit with 0
-    mkdir bin
-    mkdir bin/tests
-    unzip -q -o tests.zip -d ./bin/tests/Windows_NT.$perfArch.$perfConfig || exit 0
-    echo "unzip tests to ./bin/tests/Windows_NT.$perfArch.$perfConfig"
+    # If the tests don't already exist, download them.
+    if [ ! -d "bin" ]; then
```

Comment:
We should just be able to get rid entirely of the test unzipping here as this is handled by unstashing `nt-${arch}-${pgo}${baselineString}-test-artifacts`, yes?

Reply:
For these legs, yes,  but not for the legs that still exist in perf.groovy that we need working. Also for when people run locally.

Reply:
Yup, that makes sense.